frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -255,3 +255,43 @@ Feature: Pipeline Runs
              Then user will see the pipeline run name with failed status
               And user will see failure message below pipeline runs
               And user will also see the log snippet
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with different Workspaces
+            Given pipeline "pipeline-workspace" with PVC "git-PVC" workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-workspace"
+              And user navigates to Workspaces section
+              And user clicks on dropdown with PVC selected by default to check options
+             Then user sees option Empty Directory, Config Map, Secret, PersistentVolumeClaim, VolumeClaimTemplate
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with Empty directory
+            Given pipeline "pipeline-no-workspace" with Empty Directory is present
```

Comment:
I have no idea what "with Empty Directory is present" means. Not sure how we'd convert this into Cypress code 🤔

Reply:
We can keep it for manual execution. How do you want me to put it?

Reply:
I suggested a loose phrase earlier before this change.

> I think you mean no previous runs?
[comment](https://github.com/openshift/console/pull/8361#discussion_r598945900)

But if you want me to write out the answer, how about:
```suggestion
            Given pipeline "pipeline-no-workspace" with at least one workspace and no previous Pipeline Runs
```

Reply:
Thanks, I've updated it